README.md

```diff
@@ -1 +1,205 @@
-# to-do
\ No newline at end of file
+
+# To-Do-List-App
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+  <h3><b> To-Do-List-App Project</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To-Do-List-App <a name="about-project"></a>
+
+***To-Do-List-App***  is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. This is a simple website that allows for doing that, and I have done the project using ES6 and Webpack!
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML.
+- CSS.
+- JavaScript
+- LINTERS.
+- Webpack
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **To do list app structure**
+
+<!--![Screenshot](./assets/screenshot.jpg) -->
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- You can find the website [here](https://ochiengsteven.github.io/To-Do-List-App/)
+
```

Comment:
| Live Demo Version |
|-|
| ![image](https://github.com/Ochiengsteven/to-do/assets/93039359/c768834f-997b-4a65-b2b0-5caa599442fb) |
- [ ] It would be nice if you make this live link work because right now it is broken 😨 Kindly fix this link to make your readme descriptive and professional 👍